plenary/plenary-21.yaml

```diff
@@ -0,0 +1,287 @@
+    identifier: 241
+
+    approvals:
+      - type: affirmative
+        degree: unanimous
+        message: Approved
+
+    actions:
+      - type: resolves
+        date_effective: 2003-09-12
+        message: ISO/TC 154 resolves to endorse the resolutions and action items adopted by the ISO7372MA documented in documents 154N446 and 154N452
+
+
+  - dates:
+      - 2003-09-12
+    subject: ISO/TC 154 Processes, data elements and documents in commerce, industry and administration
+    title: "OASIS/UBL TC"
+    identifier: 242A
+
+    approvals:
+      - type: affirmative
+        degree: unanimous
+        message: Approved
+
+    actions:
+      - type: endorses
+        date_effective: 2003-09-12
+        message: Recalling the existing ISO/TC154 liaison with OASIS (see doc. 154N406, 2002-02-27), ISO/TC154 endorses the Resolution of the ISO7372MA to include the OASIS UBL TC as member (see doc. 154n452).
+
+
+  - dates:
+      - 2003-09-12
+    subject: ISO/TC 154 Processes, data elements and documents in commerce, industry and administration
+    title: "Open Applications Group (OAGi)"
+    identifier: 242B
+
+    approvals:
+      - type: affirmative
+        degree: unanimous
+        message: Approved
+
+    actions:
+      - type: resolves
+        date_effective: 2003-09-12
+        message: |
+        ISO/TC154 resolves (unanimous agreement of P-members present at our 21st session) to establish a Liaison A (active) with OAGi (see doc. 154N457 and http://www.openapplications.org/ ), with the following OAGi contact persons:
+        - Liaison to ISO TC154: Garret Minakawa, mailto:garret.minakawa@oracle.com
+        - CEO: David M. Connelly, mailto:dconnelly@openapplications.org
+        - Chief Architect: Mike Rowell, mailto:mrowell@openapplications.org
+
+
+  - dates:
+      - 2003-09-12
+    subject: ISO/TC 154 Processes, data elements and documents in commerce, industry and administration
+    title: "ISO8601:2000 => 200x (in-depth revision) date and time"
+    identifier: 243
+
+    approvals:
+      - type: affirmative
+        degree: unanimous
+        message: Approved
+
+    actions:
+      - type: notes
+        date_effective: 2003-09-12
+        message: notes that the in-depth revision project has been launched in 2000-07; as the 3 years time-frame is over, and as the draft revision is not yet ready for balloting, this work item has been withdrawn last month (2003-07)
+      - type: notes
+        date_effective: 2003-09-12
+        message: notes that the next "ISO periodic review" will take place in 2005
+      - type: notes
+        date_effective: 2003-09-12
+        message: notes the on-going work (see also Resolution 230 and Action 15 above, both open) managed by Mr. L. Visser (NEN/NL): Thanks a lot, Louis !
+      - type: confirms
+        date_effective: 2003-09-12
+        message: confirms its availability to launch a ballot on this draft revision, when available
```

Comment:
Here, I used the words `notes` and `confirms` as action types.

Reply:
Agree.